src/librustdoc/html/render/mod.rs

```diff
@@ -3683,31 +3690,30 @@ fn should_render_item(item: &clean::Item, deref_mut_: bool) -> bool {
     }
 }
 
-fn spotlight_decl(decl: &clean::FnDecl) -> String {
+fn spotlight_decl(decl: &clean::FnDecl, c: &Cache) -> String {
```

Comment:
Nit: Any reason this is just `c`, while everywhere else is was added as `cache`?

Reply:
Oh no, someone spotted my lazyness! :fearful: